### PR TITLE
Adding Australian BioCommons to the contributor list

### PR DESCRIPTION
**Pull request by ziadbkh**

### CODE REVIEW

File: public/images/contributors/white/Australian-Biocommons.svg
Comment (christopher-hakkaart) [inline public/images/contributors/white/Australian-Biocommons.svg:1]: Is it possible to make the hexagons white as well?

Comment (ziadbkh) [reply public/images/contributors/white/Australian-Biocommons.svg:1]: There seems to be something wrong with one of the tests. I tested by going back to the original file that passed the tests 2 days ago and it failed today. 
The last uploaded logo is completely white as you requested.

Comment (christopher-hakkaart) [reply public/images/contributors/white/Australian-Biocommons.svg:1]: Logo looks good. I'm not sure why it's failing. I'll try and work it out.

**Review by mashehu — APPROVED** [review]

File: public/images/contributors/white/Australian-Biocommons.svg
Comment (ziadbkh) [reply public/images/contributors/white/Australian-Biocommons.svg:1]: @christopher-hakkaart: for some reason, the marker on the map does not appear for AusBioCommons. I belive the yaml file is correct.

**Pull request merged**